### PR TITLE
feat(#1493). Add `<error>` to `XMIR` if redundant parentheses are found.

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
The `</error>` with `severity=warning` was added to `XMIR` if redundant parentheses are found.

Closes: #1493

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@Graur please, check

**Review by Graur — APPROVED** [review]
@volodya-lombrozo thanks!

**Comment by volodya-lombrozo** [comment]
@yegor256 merge please

File: eo-parser/src/main/java/org/eolang/parser/XeListener.java [hunk 1]
```diff
@@ -180,7 +186,15 @@ public void enterObject(final ProgramParser.ObjectContext ctx) {
             if (application.suffix() != null) {
                 application = application.application();
             }
-            this.check.accept(application.getText());
+            final String text = application.getText();
+            if (this.check.test(text)) {
+                this.errors.xpath("/program/errors")
```
Comment (yegor256) [inline eo-parser/src/main/java/org/eolang/parser/XeListener.java:191]: @volodya-lombrozo a more elegant and shorter solution:

`this.dirs.push().xpath("/program/errors").add("error")....pop()`

Comment (volodya-lombrozo) [reply eo-parser/src/main/java/org/eolang/parser/XeListener.java]: @yegor256 Works like a charm. Thank you. Fixed.

**Comment by volodya-lombrozo** [comment]
@yegor256 Please, merge.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**